packages/action-menu/src/ActionMenu.ts

```diff
@@ -55,6 +56,7 @@ export class ActionMenu extends ObserveSlotText(PickerBase, 'label') {
                     <sp-icon-more class="icon"></sp-icon-more>
                 </slot>
                 <slot name="label" ?hidden=${!this.hasLabel}></slot>
+                <slot name="tooltip" ?hidden=${this.open}></slot>
```

Comment:
I do not believe this is necessary. As noted, we'll address the Tooltip functionality in Overlay V2 as part of the Tooltip in that work.

Reply:
This is temporary fix. Once this issue will be addressed in Tooltip functionality in Overlay V. I will remove it.